frontend/src/services/web3-provider/walletconnect-provider.ts

```diff
@@ -1,44 +1,64 @@
 import { hexValue } from 'ethers/lib/utils';
 
 import { EthereumProvider } from '@/services/web3-provider/ethereum-provider';
-import type { Eip1193Provider } from '@/services/web3-provider/types';
 import { WalletConnect } from '@/services/web3-provider/util-export';
 
+const BEAMER_PROJECT_ID = import.meta.env.VITE_WALLET_CONNECT_PROJECT_ID || '';
```

Comment:
Aren't we going to get an error from the walletconnect relay without a project id?

Reply:
Yeah, but since the `VITE_WALLET_CONNECT_PROJECT_ID` exists in all the env files, it shouldn't be a problem.
Unless we remove them at one point, but that is highly unlikely unless we stop supporting WC.